pitch.md

```diff
@@ -23,38 +35,187 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+> !C|mb|p|mb|2C|an|Stayman --   |mb|p|mb|2H|an|2-5 !C; 2-5 !D; 4-5 !H; 2-4 !S;
+> 15-17 HC|mb|p|mb|4H|an|4+ !H; 10-13 total points |mb|p|mb|p|mb|p|pg||pc|C2|pc|
+> C5|pc|CK|pc|C3|pg||pc|S7|pc|SA|pc|S6|pc|S3|pg||pc|C4|pc|C6|pc|CA|pc|CJ|pg||pc|
+> D3|pc|DQ|pc|DK|pc|DA|pg||pc|SQ|pc|S4|pc|ST|pc|SK|pg||pc|H2|pc|H4|pc|HA|pc|H5|p
+> g||pc|H3|pc|HT|pc|HK|pc|D4|pg||pc|CQ|pc|C7|pc|S9|pc|C8|pg||pc|DT|pc|DJ|pc|D7|p
+> c|D5|pg||pc|S2|pc|HQ|pc|C9|pc|S5|pg||pc|D8|pc|H8|pc|H9|pc|D6|pg||pc|S8|pc|SJ|p
+> c|H7|pc|CT|pg||pc|D9|pc|HJ|pc|H6|pc|D2|pg||
+
+I think this could be more readable to humans.  Also I have difficulty finding
+a way to process this data without writing something myself. It is nice that
+you can do double dummy analysis and click through the tricks in the order that
+they were played if you upload a .lin file to bridgebase. The main thing that this
+doesnt offer is an easy way to share this hand with friends unless they want to
+be sent the file and then upload it to bridgebase.  I would like to have a way
+to output in an easy to read manner the hand and send it via email or social
+media to my friends without having to force them to make a bridgebase account
+and figure out how it works and without going through the clunky procedure of
+screenshotting a hand.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think it will probably be something like 2/3 of my time on design and 1/3 on
+implementation with the exception of the part of the project to do the analysis
+of the bridge hand.  I think doing the double dummy analysis will most likely
+require a large amount of implementation work.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this project has a moderate scope. At it's simplest it is quite simple.
+Just put in bids in cards in some order and output them in a nicer format. However,
+there are many more features for outputting and a few more descriptions that can
+be input that I have described already that I think would be very useful to implement.
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+
+This is a good project because it is in a domain I care about, it lends itself to
+a language as can be seen by the verbal DSL associated with describing bridge hands
+and cardplay, and the only existing DSL that does something similar has some 
+limitations that my proposed DSL would overcome. One challenge I expect to face is
+communicating about bridge to my classmates so that they can give me meaningful and
```

Comment:
This is a good point that the majority of us (I think) are not familiar with bridge. I think that as the project progresses, it could be interesting to trail the app on us, the classmates, to see how effective of a teaching tool it can be, especially because you noted that as one focus in your domain is using this information communication as a teaching tool.